cmake/modules/BuildJaeger.cmake

```diff
@@ -44,11 +44,11 @@ function(build_jaeger)
 
   if(CMAKE_MAKE_PROGRAM MATCHES "make")
     # try to inherit command line arguments passed by parent "make" job
-    set(make_cmd $(MAKE))
+    set(make_cmd $(MAKE) Jaeger)
```

Comment:
> nature of ninja build differs from make a bit, small fixes to make jaeger compatible with ninja

this is wrong. the problem in existing code is that `make_cmd` is not used at all. and `$(MAKE)` should not be used unconditionally.

the idea of using `make_cmd` was to use the make server which is missing in, for instance, ninja. and we should fall back to `cmake` if `make` is not used as the generator.

Reply:
> this is wrong. the problem in existing code is that `make_cmd` is not used at all. 

@tchaikov I see, won't https://github.com/ceph/ceph/pull/38783/files/bd0ca9dcb85f317555d7306c2bf7beab2cb54406..dcf08bcb6366255c8590c6fc5d54ac9bb92e6351#diff-c9968a694423886ad4fa52ca9edfd0c43ca751ea08733b3383d32efc0a14159eR61 be using `make_cmd`  ?

Reply:
sorry, i was referencing `INSTALL_COMMAND` before the change, and i think the commit message is not accurate.

Reply:
please do not use "build/op" as the prefix in the title, it's not specific enough. cmake would suffice.

Reply:
this comment is not addressed.

> buildlib.cmake

we don't have this file or directory.

> nature of ninja build differs from make a bit, small fixes to make jaeger compatible with ninja

this is still wrong.

Reply:
> nature of ninja build differs from make a bit, small fixes to make jaeger compatible with ninja
removed and updated to mention exact changes and how it affects our builds: https://github.com/ceph/ceph/pull/38783/commits/df1add80b690cab0cc2291bda786b9fc9b351619
kept `cmake` alone in the title